src/doc/reference/glossary/serial_semantics.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Serial semantics
+tags: serial semantics
+---
+The behavior of a Cilk program when executed as the
+{% defn "serialization" %} of the program.
```

Comment:
Since the "serialization" entry just points to "serial projection," should this reference be "serial projection," instead of "serialization"?